pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -33,7 +33,17 @@ func WithAuditPolicies(targetName string, targetNamespace string, assetDelegateF
 	}
 }
 
-// getRawAuditPolicies returns a raw config map that holds the audit policies for the target namespaces
+// GetAuditPolicies returns a config map that holds the audit policies for the target namespaces and name
+func GetAuditPolicies(targetName, targetNamespace string) (*corev1.ConfigMap, error) {
+	rawAuditPolicies, err := getRawAuditPolicies(targetName, targetNamespace)
+	if err != nil {
+		return nil, err
+	}
+
+	return resourceread.ReadConfigMapV1OrDie(rawAuditPolicies), nil
```

Comment:
do we have a `ReadConfigMapV1` version that returns an `error`?

Reply:
I’m not aware of any, but this is a common practice. It is considered a programmer error when it fails